backend/tests/conftest.py

```diff
@@ -528,21 +528,29 @@ def pytest_collection_modifyitems(config, items):
                 item.add_marker(skip_api)
 
 
-def setup_deployment() -> dict[str, str]:
+def setup_deployment(test_name) -> dict[str, str]:
     deployer = os.getenv("CONTRACTS_DEPLOYER_URL")
-    testname = f"octant_test_{random_string()}"
+    env_name = f"octant_test_{random_string()}"
+    print(f"test {test_name}, environment name: {env_name}")
```

Comment:
You can consider replacing print with logging, exactly the same as we do in: backend/tests/conftest.py:675. Generally it's considered as a better practice.

Reply:
Since this code is being executed before flask app creation, referencing current_app leads to "execution out of app context".

Reply:
Done. Prints where there because current_app.logger was not available, because this code is executed before flask app is created. Replaced prints with imported and configured logger.